app/presenters/document/field_group_presenter.rb

```diff
@@ -169,5 +163,25 @@ def section_nested_hash(mappings, subject = document)
         end
       end
     end
+
+    private
+
+    def entity_section?(section)
+      section[:entity_name] == 'agents' && section[:entity_proxy_field] == 'dcCreator'
+    end
+
+    def entity_url(default_url)
+      result = default_url
+      about = @document.fetch('agents.about')
+      if about
+        # about => http://data.europeana.eu/:type/:namespace/:identifier, where :type MUST be 'agent'
```

Comment:
Also, and more importantly, the hostname in the agent's `about` field URI must be "data.europeana.eu"